pkgs/by-name/wa/wasabiwallet/package.nix

```diff
@@ -27,18 +27,18 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "wasabiwallet";
-  version = "2.0.8.1";
+  version = "2.5.1";
 
   src = fetchurl {
-    url = "https://github.com/zkSNACKs/WalletWasabi/releases/download/v${version}/Wasabi-${version}.tar.gz";
-    sha256 = "sha256-9q93C8Q4MKrpvAs6cb4sgo3PDVhk9ZExeHIZ9Qm8P2w=";
+    url = "https://github.com/zkSNACKs/WalletWasabi/releases/download/v${version}/Wasabi-${version}-linux-x64.tar.gz";
```

Comment:
```suggestion
    url = "https://github.com/WalletWasabi/WalletWasabi/releases/download/v${version}/Wasabi-${version}-linux-x64.tar.gz";
```

Reply:
I didn't changed the organization because of reviewer could result suspicious but yes, it is the correct thing to do.